.github/workflows/matlab.yml

```
@@ -37,7 +37,7 @@ concurrency:
 
 jobs:
 
-  matlab:
+  matlab-ubuntu:
```

How about removing `matlab-` prefix because this file is only for MATLAB?

```suggestion
  ubuntu:
```